scripts/firedrake-zenodo

```diff
@@ -115,7 +116,9 @@ parser.add_argument("--ignore-existing-records", action="store_true",
                     help="When creating Zenodo meta-record, ignore any existing records which match the tag")
 parser.add_argument("--skip-missing", action="store_true",
                     help="When creating Zenodo meta-record, skip missing components?")
-
+# TODO: Backward compatibility. Remove this option eventually.
+parser.add_argument("--ignore-src-petsc4py", action="store_true",
+                    help="Ignore src/petsc4py? (The latest Firedrake builds petsc4py from petsc source tree (not from src/petsc4py).")
```

Comment:
Don't do this. Instead move `petsc4py` to `petsc4py_old`.

Reply:
Done.